migration/migrator/migrations/course/20240219030249_peer_panel_graders.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE electronic_gradeable ADD COLUMN eg_peer_panel VARCHAR(5) DEFAULT '11111';")
```

Comment:
Essentially storing data as a bit string seems rather prone to error, and will be challenging to maintain in the future if additional panels get added.  Creating a new table with a foreign-key relationship to the `electronic_gradeable` column is probably the best way to go about doing this.

Reply:
yes, you are right 
it will be tedious to append a default character to the string for each of the gradeable out there, if a new panel is added.
I'll soon implement the foreign-key relation which you have mentioned
And I plan to use Boolean datatype in new table for each of the column instead of binary
is it ok ?

Reply:
Using a boolean would be great.